src/sources/git.ts

```diff
@@ -22,25 +28,26 @@ async function download(name: string, version: string, source: GitService | unde
 	await fse.unlink(fileName);
 } // }}}
 
-export async function install(extensionName: string, extensionVersion: string | undefined, source: GitService | undefined, config: GitConfig, temporaryDir: string, enabled: boolean, debugChannel: vscode.OutputChannel | undefined): Promise<InstallResult> { // {{{
-	const results = await got.get(config.getReleasesUrl(extensionName, source), config.getHeaders(source)).json();
+async function findLatestAsset(extensionName: string, source: GitService | undefined, config: GitConfig, targetVersion?: string): Promise<AssetInfo | undefined> {
+	const releases = await got.get(config.getReleasesUrl(extensionName, source), config.getHeaders(source)).json();
 
-	if(!results || !Array.isArray(results)) {
-		return;
+	if(!releases || !Array.isArray(releases)) {
+		return undefined;
 	}
 
 	let name: string = '';
 	let version: string = '';
 	let url: string = '';
 
-	for(const result of results) {
-		if(result.assets) {
-			for(const asset of result.assets) {
+	for(const release of releases) {
+		if(release.assets) {
+			for(const asset of release.assets) {
 				const match = /^(.*?)-(\d+\.\d+\.\d+)\.vsix$/.exec(asset.name as string);
 
 				if(match) {
-					if(extensionVersion) {
-						if(extensionVersion === match[2]) {
+					if(targetVersion) {
+						// Looking for a specific version
+						if(targetVersion === match[2]) {
 							name = match[1];
 							version = match[2];
 							url = asset.browser_download_url as string;
```

Comment:
Can you also use `config.getAssetUrl()` on this line

Reply:
Ah, good catch! Didn't test this case.☹️